src/plugins/intel_cpu/src/utils/precision_support.cpp

```diff
@@ -34,7 +34,7 @@ static bool hasFP16HardwareSupport(const ov::element::Type& precision) {
 
 static bool hasBF16HardwareSupport(const ov::element::Type& precision) {
 #if defined(OPENVINO_ARCH_X86_64)
-        if (dnnl::impl::cpu::x64::mayiuse(dnnl::impl::cpu::x64::avx512_core) ||
+        if (dnnl::impl::cpu::x64::mayiuse(dnnl::impl::cpu::x64::avx512_core_bf16) ||
```

Comment:
That is not correct change. We support BF16 on systems with avx512_core only for debbuging/testing purposes

Reply:
reverted changes